src/components/Wallet/ConfirmWallet.jsx

```diff
@@ -25,6 +27,11 @@ class WalletConfirmation extends React.Component {
           </TableHead>
           <TableBody>{this.renderConfirmationInfo()}</TableBody>
         </Table>
+        {startingAddressIndex > 0 && (
+          <Typography variant="subtitle1">
+            Attention: Starting Address Index set to {startingAddressIndex}
+          </Typography>
```

Comment:
To me this looks odd:

![gscreenshot_2020-07-02-145856](https://user-images.githubusercontent.com/4116405/86399091-a6ddbf80-bc74-11ea-9259-4c76b23ab38c.png)

I think we should use `<Alert severity="info">` or `<Alert severity="warning">` instead.  @bucko13 thoughts?

![gscreenshot_2020-07-02-145828](https://user-images.githubusercontent.com/4116405/86399097-ab09dd00-bc74-11ea-8c09-69ca92130e22.png)



See also https://material-ui.com/components/alert/

Reply:
Agreed, I think I like Alert Info.

Reply:
So the idea would be info alert if it's a custom start index and nothing if it's default or for all circumstances? 

For a future update, I kind of like the idea of using a chip collection for the other items in the config (network, index, address type) could be a nice way to be able to display this type of info. https://material-ui.com/components/chips/

Reply:
> So the idea would be info alert if it's a custom start index and nothing if it's default or for all circumstances?

Yeah, exactly.

> For a future update, I kind of like the idea of using a chip collection for the other items in the config (network, index, address type) could be a nice way to be able to display this type of info. https://material-ui.com/components/chips/

Agreed.

Reply:
done.